torch/csrc/distributed/rpc/init.cpp

```diff
@@ -122,6 +122,11 @@ PyObject* rpc_init(PyObject* /* unused */) {
           offers best-effort error detection, and applications should not use
           ``UserRRef``s after ``rpc.shutdown()``.
 
+          .. warning::
+              RRefs can only be serialized and deserialized by the RPC module.
+              Serializing and deserializing RRefs without RPC will lead to
+              errors.
+
```

Comment:
Should we add some more details here mentioning that if we use torch.save(), jit.pickle() or python pickle on RRefs this would cause unintended consequences?

Reply:
Also, I'm assuming we will remove this warning once https://github.com/pytorch/pytorch/pull/34689 and https://github.com/pytorch/pytorch/pull/34755 lands?

Reply:
> Should we add some more details here mentioning that if we use torch.save(), jit.pickle() or python pickle on RRefs this would cause unintended consequences?

Yes, let me add. 

> Also, I'm assuming we will remove this warning once #34689 and #34755 lands?

It might be better keeping this warning even after we land those two? Because otherwise, applications would only learn that through trial-error?